src/SchemaUtils.ts

```diff
@@ -249,12 +264,46 @@ function parseStepWithSelectors(type: string, step: object): StepWithSelectors {
   };
 }
 
+function parseClickAttributes(step: object): ClickAttributes {
+  const attributes: ClickAttributes = {
+    offsetX: parseNumber(step, 'offsetX'),
+    offsetY: parseNumber(step, 'offsetY'),
+  };
+  const deviceType = parseOptionalString(step, 'deviceType');
+  if (deviceType && !pointerDeviceTypes.has(deviceType)) {
```

Comment:
nit: could we write a function for this which guards the type of deviceType? E.g., `isKnownDeviceType` similar to, e.g., how `isIntegerArray` works.

Reply:
Done.